docs/designs/ruby_test_plan_002.md

```diff
@@ -0,0 +1,112 @@
+# Test plan for RubyGems/Bundler package manager
+
+This document is intended to describe RubyGems/Bundler package manager
+usage patterns to be tested. The document follows package manager design
+proposed in [PR-565](https://github.com/containerbuildsystem/cachi2/pull/565).
+The document captures behaviors under test in a DSL for test scenario
+description. The format is straightforward and self-documenting, for a formal
+specification and tooling please refer to [the specification](https://cucumber.io/docs/gherkin/reference/).
+
+## General overview
+
+Cachi2 is intended to prefetch project's dependencies and to prepare a local
+environment to make it possible to build projects without network connectivity.
+Cachi2 can handle a number of package formats, RubyGems/Bundler (RG/B) is among
+the supported ones. Cachi2 does not support all features of RG/B, opting to
+support the most widespread and safe ones instead of supporting everything.
+
+The tests are to check that conforming packages could be preprocessed by Cachi2
+and that non-conforming packages are rejected by Cachi2.
+
+The checks are expected to be run against actual packages published either on
+rubygems.org, on github.com or locally.
+
+Each example in every test scenario requires a separate repository/package
+demonstrating an aspect under test. A list of necessary repositories could be
+found [below](#necessary-repositories).
+
+## Test cases
```

Comment:
Will we add a scenario with multiple packages in a single repo ?

Reply:
Do you mean e2e or integration test? For integration test in `Cachi2 can preset configuration for hermetic builds` feature `a user prepares a repository with a Ruby project` scenario is defined for both single project and multi-project repositories.